core/src/main/java/hudson/cli/GroovyshCommand.java

```diff
@@ -60,7 +60,7 @@ public String getShortDescription() {
     @Override
     protected int run() {
         // this allows the caller to manipulate the JVM state, so require the admin privilege.
-        Jenkins.get().checkPermission(Jenkins.RUN_SCRIPTS);
+        Jenkins.getActiveInstance().checkPermission(Jenkins.ADMINISTER);
```

Comment:
and this active instance?

Reply:
I believe this branch originated before the removal of the deprecated method was in master, fixed now